### DISCUSSION TITLE
Fan Control and GPU boost Control on TUF A15 2021

### DISCUSSION THREAD

**Discussion by nilaysriv** [seerge/g-helper#2471] [Q&A] [answered]
First of all, I want to thank you for the awesome program and support you've given for our devices. 
I tried the build from this discussion: https://github.com/seerge/g-helper/discussions/2272
and manual fan controls were surprisingly working.
I tried the latest build today and that option seems to have been removed.
I was wondering if it would be added back again?
Also, is there any possibility to add GPU Dynamic boost control on 2021 model like the newer one? I want to keep the full 25W boost for turbo mode and have a bit better thermals (albeit the performance loss) on balanced mode.
Thank you again. Have a good day! @seerge

**Comment by seerge** [comment]
@nilaysriv hello,

Experimental manual fan controls are not part of the main releases. I haven't removed anything there, you can still use it. I'm periodically updating experimental build as well, and add recent changes from main branch https://github.com/seerge/g-helper/discussions/2272

It's not possible to add Dynamic Boost for old devices, as it needs to be implemented by Asus and supported by BIOS.

**Reply by nilaysriv** [reply]
Hey! Right now I went back to the build with manual fan control. But just to be sure, future experimental builds will carry all the pre-release features like these, right?
My apologies, I am not very good at navigating github but do these qualify for experimental features?
![image_2024-04-22_000036942](https://github.com/seerge/g-helper/assets/76720071/ff727ae6-aa6b-4601-a4f0-831984134d79)

**Reply by seerge** [reply, marked as answer]
@nilaysriv yes, i will try to keep them up to date.

**Reply by nilaysriv** [reply]
Thank you! Really appreciate all the help. Cheers!
